symplyphysics/core/convert.py

```diff
@@ -13,3 +13,7 @@ def convert_to(value: SymQuantity, target_unit: SymQuantity) -> Expr:
     assert_equivalent_dimension(value, value.dimension.name, "convert_to",
         target_quantity.dimension)
     return sympify(value.scale_factor) * (1 / sympify(target_quantity.scale_factor))
+
+
+def convert_to_float(value: SymQuantity) -> float:
```

Comment:
I suggest to also accept `Expr` type of value with automatic conversion to `Quantity`. This way you'll be able to shorten code.

Reply:
Do you mean allowing to write things like `result = law.subs(...); return convert_to_float(result)`, omitting the `Quantity(result)` part in the end? The thing is that `convert_to` accepts only `Quantities` strictly, because it uses the `dimension` field which is not present in `Mul` and other classes.

Reply:
Yes, but you can see, that we explicitly convert to Quantity almost everywhere. You can do it in your helper function, when you get input parameter as `Expr`

Reply:
I know, I just wanted to make sure I understand you